std/format.d

```diff
@@ -2921,7 +2922,7 @@ if (is(T == class) && !is(T == enum))
         {
           //string delegate() dg = &val.toString;
             Object o = val;     // workaround
-            string delegate() dg = &o.toString;
+            const string delegate() dg = &o.toString;
```

Comment:
IIRC, there are bugs with `const` and delegates. I wouldn't advise mixing them. And if that's supposed to be `const(string)` as opposed to making the delegate itself `const` (I honestly don't know which it's supposed to do here), then that wouldn't match `toString` on `Object`. I'd just leave this one out, particularly since odds are that it doesn't buy us anything anyway.